packages/components/src/interfaces/complex/DynamicDataBanner.ts

```diff
@@ -10,7 +10,7 @@ export const DYNAMIC_DATA_BANNER_TYPE = "dynamicdatabanner"
 // Hardcoded for now because
 // 1. MUIS is the only use case and there's always 6 prayer timeslots
 // 2. No other known use cases have been identified
-export const NUMBER_OF_DATA = 6
+export const DYNAMIC_DATA_BANNER_NUMBER_OF_DATA = 6
```

Comment:
purpose: to improve clarity as `NUMBER_OF_DATA` can be misleading, esp. when i started working on this PR and we were considering giving users the option to determine 3/6 pages to display